applications/NXmx.nxdl.xml

```diff
@@ -666,6 +666,12 @@
                         </doc>
                     </field>
 
+                    <field name="bit_depth_image" type="NX_POSINT" recommended="true">
```

Comment:
`image_bit_depth` is clearer and better fits the naming patterns of the other fields.